internal/db/schema/migrations/oss/postgres/70/02_sessions.up.sql

```diff
@@ -0,0 +1,162 @@
+begin;
+
+  -- Replaces trigger from 56/02_add_data_key_foreign_key_references.up.sql
+  drop trigger cancel_session_with_null_fk on session;
+  create or replace function cancel_session_with_null_fk() returns trigger
```

Comment:
This should just be `create` without the `or replace`.

Reply:
Should I manually drop the function first if I remove the replace statement? Otherwise the database migration will return an error for using the same function name twice.